src/platforms/dart/common/performance/instrumentation/automatic-instrumentation.mdx

```diff
@@ -31,3 +31,15 @@ The span finishes once the request has been executed. The span `status` depends
 When the HTTP request throws an `Exception`, Sentry's SDK associates this exception to the running span. If you haven't set the SDK to swallow the exception and capture it, the span and `SentryEvent` will be linked when viewing it on the **Issue Details** page in [sentry.io](https://sentry.io).
 
 Learn more in our [Dio integration documentation](/platforms/dart/configuration/integrations/dio/#performance-monitoring-for-http-requests).
+
+### File I/O Instrumentation
+
+The Sentry-specific file I/O implementation starts a span out of the active span, bound to the scope for each file I/O operation. The SDK sets the span `operation` to `file.copy`/`file.write`/`file.delete`/`file.open`/`file.read`/`file.rename` and `description` to `filename`, e.g. `file.txt`
```

Comment:
@AbhiPrasad what do you think about all these operations? I still can change it but Dart has specific methods per operation and if I don't do the operation suffix, everything would be `write/read` and you can't really tell which operation was executed.

Reply:
cc @romtsn

Reply:
I think these operations are fine.

Reply:
@marandaneto could you also update this page please? https://develop.sentry.dev/sdk/performance/span-operations/#mobile